<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QImageAdjustSet</class>
 <widget class="QDialog" name="QImageAdjustSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>260</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>260</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>250</width>
    <height>260</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="FuncPages">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="ImageSizeSet">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="H1">
         <item>
          <widget class="QLabel" name="BasisLabel">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);
font-size:13px;</string>
           </property>
           <property name="text">
            <string>调整依据:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="SetPixel">
           <property name="minimumSize">
            <size>
             <width>65</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>65</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);
font-size:13px;</string>
           </property>
           <property name="text">
            <string>像素</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="SetPercentage">
           <property name="minimumSize">
            <size>
             <width>65</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>65</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);
font-size:13px;</string>
           </property>
           <property name="text">
            <string>百分比</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="H2">
         <item>
          <widget class="QLabel" name="VerticalIcon">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="resource.qrc">:/LoginUi/prefix1/Resource/EduFunc/HorizontalSize.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="VerticalTitle">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);
font-size:13px;</string>
           </property>
           <property name="text">
            <string>水平(W):</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="VerticalValue">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QSpinBox{
background-color: rgb(200, 200, 200);
color:rgb(0, 0, 0);
font-size:13px;
border-width:3px;
border-color:#c0c0c0;
}
</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="H3">
         <item>
          <widget class="QLabel" name="HorizontalIcon">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="resource.qrc">:/LoginUi/prefix1/Resource/EduFunc/VerticalSize.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="HorizontalTitle">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);
font-size:13px;</string>
           </property>
           <property name="text">
            <string>垂直(H):</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="HorizontalValue">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QSpinBox{
background-color: rgb(200, 200, 200);
color:rgb(0, 0, 0);
font-size:13px;
border-width:3px;
border-color:#c0c0c0;
}
</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="H5">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>66</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QCheckBox" name="AspectRatio">
           <property name="minimumSize">
            <size>
             <width>130</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>130</width>
             <height>20</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);
font-size:13px;</string>
           </property>
           <property name="text">
            <string>保持纵横比</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="H4">
         <property name="spacing">
          <number>15</number>
         </property>
         <property name="rightMargin">
          <number>9</number>
         </property>
         <item>
          <spacer name="S2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="SizeOk">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>25</height>
            </size>
           </property>
           <property name="mouseTracking">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);
font-size:13px;</string>
           </property>
           <property name="text">
            <string>OK</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="SizeCancel">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);
font-size:13px;</string>
           </property>
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="ImageAngleSet">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <spacer name="S8">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>48</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="H6">
         <item>
          <widget class="QLabel" name="AngleTitle">
           <property name="minimumSize">
            <size>
             <width>115</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>115</width>
             <height>20</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);
font-size:13px;
font: Bold -75;</string>
           </property>
           <property name="text">
            <string>请输入旋转角度:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="S4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="H7">
         <property name="spacing">
          <number>20</number>
         </property>
         <property name="leftMargin">
          <number>10</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="resource.qrc">:/LoginUi/prefix1/Resource/EduFunc/Spin.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="AngleValue">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QSpinBox{
background-color: rgb(200, 200, 200);
color:rgb(0, 0, 0);
font-size:13px;
border-width:3px;
border-color:#c0c0c0;
}
</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="S9">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>47</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="H8">
         <item>
          <spacer name="S7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="AngleOk">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>25</height>
            </size>
           </property>
           <property name="mouseTracking">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);
font-size:13px;</string>
           </property>
           <property name="text">
            <string>OK</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="AngleCancel">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);
font-size:13px;</string>
           </property>
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>SizeOk</sender>
   <signal>clicked()</signal>
   <receiver>QImageAdjustSet</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>111</x>
     <y>218</y>
    </hint>
    <hint type="destinationlabel">
     <x>124</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SizeCancel</sender>
   <signal>clicked()</signal>
   <receiver>QImageAdjustSet</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>218</y>
    </hint>
    <hint type="destinationlabel">
     <x>124</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AngleOk</sender>
   <signal>clicked()</signal>
   <receiver>QImageAdjustSet</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>128</x>
     <y>237</y>
    </hint>
    <hint type="destinationlabel">
     <x>124</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AngleCancel</sender>
   <signal>clicked()</signal>
   <receiver>QImageAdjustSet</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>237</y>
    </hint>
    <hint type="destinationlabel">
     <x>124</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
